fr.nytuo.cosmiccomics.metainfo.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>fr.nytuo.cosmiccomics</id>
+  
+  <name>Cosmic Comics</name>
+  <summary>Comics and manga reader and collectionner</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+
+  <content_rating type="oars-1.1" />
+
+  <developer id="nytuo">
+    <name>Arnaud BEUX</name>
+  </developer>

```

Comment:
Use `developer_name`, Flathub does not yet support developer tag

Reply:
In the doc the developer_name tag is deprecated so I've change it.
i'll revert it to a developer_name tag then